app/templates/components/upcoming-project-card.hbs

```diff
@@ -0,0 +1,130 @@
+        <span data-test-hearings-waived-message="{{project.id}}">You have opted out of submitting hearings</span>
+      {{/if}}
+      {{#if (and hearingsSubmitted (not hearingsWaived))}}
+        {{#deduped-hearings-list project=project as |dedupedHearings|}}
+          <h6>Comminity Board Hearings:</h6>
+              <ul class="no-bullet">
+              {{#each dedupedHearings as |hearing|}}
+                <li class="grid-x small-margin-bottom">
+                  <div class="cell shrink small-margin-right">
+                    {{#if (is-before hearing.dcpDateofpublichearing)}}
+                      {{fa-icon "calendar" class="light-gray" fixedWidth=true}}
+                    {{else}}
+                      {{fa-icon "check" class="blue" fixedWidth=true}}
+                    {{/if}}
+                  </div>
+                  <div class="cell auto">
+                    <strong data-test-hearing-location="{{hearing.id}}" class="display-block">
+                      {{~hearing.dcpPublichearinglocation~}}
+                    </strong>
+                    <span data-test-hearing-date="{{hearing.id}}">
+                      {{~moment-format hearing.dcpDateofpublichearing "MM/D/YYYY"~}}
+                    </span>
+                    <span class="light-gray">|</span>
+                    <span data-test-hearing-time="{{hearing.id}}">
+                      {{~moment-format hearing.dcpDateofpublichearing "h:mm A"~}}
+                    </span>
+                    <span data-test-hearing-actions-list="{{hearing.id}}" class="display-block text-tiny">
+                      {{#each hearing.hearingActions as |action index|}}
+                        {{if (gt index 0) ", "}}{{action.dcpUlurpnumber}}
+                      {{/each}}
+                    </span>
+                  </div>
+                </li>
+              {{/each}}
+              </ul>
+        {{/deduped-hearings-list}}
+      {{/if}}
+      {{#if (and (not hearingsSubmitted) (not hearingsWaived))}}
+        <LinkTo
+          @route="my-projects.project.hearing.add"
+          @model={{project.id}}
+          class="button expanded tiny-margin-bottom"
+          data-test-button="submitHearing"
+        >
+          Submit Comminity Board Hearing Info
+        </LinkTo>
+        <p class="text-small text-right">
+          <span class="display-inline-block">
+            <button
+              class="button tiny clear no-margin"
+              onClick={{action "openOptOutHearingPopup"}}
+              data-test-button="optOutHearingOpenPopup"
+            >
+              Opt out of hearings
+            </button>
+            {{waive-hearings-popup project=project showPopup=showPopup}}
+          </span>
+        </p>
+      {{/if}}
+    </div>
+    <div class="cell medium-4 large-shrink">
+      <p class="text-center medium-margin-right medium-margin-left">
+        {{#if (eq project.upcomingMilestonePlannedStartDate)}}
```

Comment:
if `project.upcomingMilestonePlannedStartDate` is equal to what?

Reply:
@godfreyyeung It think this is yours?